Extensions/TweenBehavior/JsExtension.js

```diff
@@ -0,0 +1,524 @@
+/**
+ * This is a declaration of an extension for GDevelop 5.
+ *
+ * ℹ️ Run `node import-GDJS-Runtime.js` (in newIDE/app/scripts) if you make any change
+ * to this extension file or to any other *.js file that you reference inside.
+ *
+ * The file must be named "JsExtension.js", otherwise GDevelop won't load it.
+ * ⚠️ If you make a change and the extension is not loaded, open the developer console
+ * and search for any errors.
+ *
+ * More information on https://github.com/4ian/GDevelop/blob/master/newIDE/README-extensions.md
+ */
+module.exports = {
+  createExtension: function (_, gd) {
+    const extension = new gd.PlatformExtension();
+    extension
+      .setExtensionInformation(
+        "Tween",
+        _("Tween animation"),
+        _("Animate object properties via tweening."),
+        "Matthias Meike",
+        "Open source (MIT License)"
+      )
+      .setExtensionHelpPath("/all-features/tween");
+
+    var tweenBehavior = new gd.BehaviorJsImplementation();
+
+    tweenBehavior.updateProperty = function (
```

Comment:
I don't need shared data or behavior properties. If I remove them GD refuses to load the extension. What is the correct way of not defining properties/shared data?

Reply:
What do you mean by "If I remove them"? Removing properties while keeping functions should work:

```
    tweenBehavior.updateProperty = function (
      behaviorContent,
      propertyName,
      newValue
    ) {
       return false;
    };

     tweenBehavior.getProperties = function (behaviorContent) {
      var behaviorProperties = new gd.MapStringPropertyDescriptor();
       return behaviorProperties;
    };

     tweenBehavior.setRawJSONContent(
      JSON.stringify({})
    );
```